tasks/src/question-answering/about.md

```diff
@@ -50,3 +50,7 @@ Would you like to learn more about QA? Awesome! Here are some curated resources
 - [PyTorch](https://github.com/huggingface/transformers/tree/main/examples/pytorch/question-answering)
 - [TensorFlow](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/question-answering)
 - [Flax](https://github.com/huggingface/transformers/tree/main/examples/flax/question-answering)
+
+### Documentation
+
```

Comment:
```suggestion
```

Reply:
Not sure why you suggest removing this blank line. a) If it is built the same way the docs are, the blank link is necessary, otherwise some formatting breaks. b) there's a blank line after all the other headers, so not sure why it shouldn't be here. Can you please explain?